libs/libxx/cxx.defs

```diff
@@ -22,3 +22,14 @@ CXXSRCS += libxx_cxa_guard.cxx libxx_cxapurevirtual.cxx
 CXXSRCS += libxx_delete.cxx libxx_delete_sized.cxx libxx_deletea.cxx
 CXXSRCS += libxx_deletea_sized.cxx libxx_new.cxx libxx_newa.cxx
 CXXSRCS += libxx_stdthrow.cxx
+
+# Note: Our implmenetations of operator new are not conforming to
+# the standard. (no bad_alloc implementation)
+#
+# libxx_new.cxx:64:11: error: 'operator new' is missing exception specification
+#       'throw(std::bad_alloc)' [-Werror,-Wmissing-exception-spec]
```

Comment:
But should we implement bad_alloc instead?

Reply:
@yamt can you open an issue to remind ourselves to fix this? shouldn't be hard